site/src/styles/global.css

```
@@ -0,0 +1,357 @@
+@import '@fontsource-variable/space-grotesk';
+@import '@fontsource/ibm-plex-sans/400.css';
+@import '@fontsource/ibm-plex-sans/500.css';
+@import '@fontsource/ibm-plex-sans/600.css';
+
+@tailwind base;
```

Biome (2.4.4)

[error] Unexpected unknown at-rule: tailwind 

(lint/suspicious/noUnknownAtRules)